doc/source/tutorials/lab.rst

```diff
@@ -611,8 +628,10 @@ in this case ``"twpa_pump"``.
     from qibolab.instruments.dummy import DummyInstrument
     from qibolab.instruments.oscillator import LocalOscillator
 
+    FOLDER = Path.cwd()
```

Comment:
```suggestion
    FOLDER = Path(__file__).absolute().parent
```
Since you never know in which directory is the person launching it.

Reply:
Right after writing this, I realized that could be the reason...

I thought a while about that, but the only two proposals I have are: 
- avoid testing or 
- leave it as it is, but add a comment explaining

It's not great to see a comment *for internal testing reasons, but you should do in this other way*. But I would leave a simpler comment anyhow 
```suggestion
    FOLDER = Path.cwd()  # Path(__file__).absolute().parent
```
at least the reader has a chance of understanding the difference.